windows/9.2.x/sitecore-xp-sxa/build.json

```diff
@@ -11,7 +11,7 @@
     {
       "tag": "sitecore-xp-sxa-cd:9.2.0-windowsservercore-${windowsservercore_version}",
       "build-options": [
-        "--build-arg BASE_IMAGE=sitecore-xp-spe-standalone:9.2.0-windowsservercore-${windowsservercore_version}",
+        "--build-arg BASE_IMAGE=sitecore-xp-cd:9.2.0-windowsservercore-${windowsservercore_version}",
```

Comment:
Wouldn't this need to refer to the SPE cd?
There are no SPE files required? - just checking...

Reply:
SPE is only present on cm/standalone and sql